test/unit/gateways/wompi_test.rb

```diff
@@ -106,18 +106,34 @@ def test_successful_refund_to_void
       @gateway.refund(@amount, @credit_card, @options)
     end.check_request do |endpoint, data, _headers|
       assert_match 'void_sync', endpoint
-      assert_match '{}', data
+      assert_match @amount.to_s, data
     end.respond_with(successful_void_response)
     assert_success response
 
     assert_equal '113879-1635301067-17128', response.authorization
     assert response.test?
   end
 
-  def test_successful_void
-    @gateway.expects(:ssl_post).returns(successful_void_response)
+  def test_successful_partial_refund_to_void
+    response = stub_comms(@gateway) do
+      @gateway.refund(@amount - 50000, @credit_card, @options)
+    end.check_request do |endpoint, data, _headers|
+      assert_match 'void_sync', endpoint
+      assert_match (@amount - 50000).to_s, data
+    end.respond_with(successful_void_response)
+    assert_success response
 
-    response = @gateway.void(@amount, @options)
+    assert_equal '113879-1635301067-17128', response.authorization
+    assert response.test?
+  end
+
+  def test_successful_void
+    response = stub_comms(@gateway) do
+      @gateway.void(@amount, @options)
+    end.check_request do |endpoint, data, _headers|
+      assert_match 'void_sync', endpoint
+      assert_match '{}', data
+    end.respond_with(successful_void_response)
```

Comment:
Would appreciate any feedback on these test cases to make sure they are doing what I think they are doing 😁